include/click/packet.hh

```diff
@@ -1265,6 +1279,8 @@ Packet::next() const
     return (Packet *)(skb()->next);
 #elif CLICK_PACKET_USE_DPDK
     return all_anno()->next;
+#elif INLINED_ALLANNO
+    return nextp;
```

Comment:
Isn't there a way to have a macro to prevent all thse ifdef? Like _aa is a typedef/#define for this? Or change "_aa." per something likge getanno() which returns this or Anno* according to the mode?

Reply:
This is actually a nice idea! packet.hh is too messy. I will try what you suggested. Thanks :)